docs/modules/ROOT/pages/contributors/naming-conventions.adoc

```diff
@@ -0,0 +1,126 @@
+= Naming Conventions
+
+This document lists the naming conventions to be applied in the different parts
+of KubeArchive codebase.
+
+== Helm Chart
+
+The Helm Chart templates are located into different directories that match the different
+components of kubearchive:
+
+* Sink
+* API Server
+* Operator
+* Database
+
+The common resources are outside these folders.
+
+The templated variables are located in [filename]`kubearchive/chars/kubearchive/values.yaml`
+following the same categories.
+
+=== Conventions
+
+* Follow the DRY (Don't Repeat Yourself) principle.
+When a value is reused extract it and reference it.
+If a part of the value is reused (like a prefix with the app name) also
```

Comment:
```suggestion
If part of the value is reused (like a prefix with the app name) also
```

Reply:
I restructured the docs so this sentence isn't there anymore